src/set_theory/ordinal/basic.lean

```diff
@@ -1231,72 +1246,56 @@ open ordinal
 /-- The ordinal corresponding to a cardinal `c` is the least ordinal
   whose cardinal is `c`. For the order-embedding version, see `ord.order_embedding`. -/
 def ord (c : cardinal) : ordinal :=
-begin
-  let ι := λ α, {r // is_well_order α r},
-  have : Π α, ι α := λ α, ⟨well_ordering_rel, by apply_instance⟩,
-  let F := λ α, ordinal.min ⟨this _⟩ (λ i:ι α, ⟦⟨α, i.1, i.2⟩⟧),
-  refine quot.lift_on c F _,
-  suffices : ∀ {α β}, α ≈ β → F α ≤ F β,
-  from λ α β h, le_antisymm (this h) (this (setoid.symm h)),
-  intros α β h, cases h with f, refine ordinal.le_min.2 (λ i, _),
-  haveI := @rel_embedding.is_well_order _ _
-    (f ⁻¹'o i.1) _ ↑(rel_iso.preimage f i.1) i.2,
-  rw ← show type (f ⁻¹'o i.1) = ⟦⟨β, i.1, i.2⟩⟧, from
-    quot.sound ⟨rel_iso.preimage f i.1⟩,
-  exact ordinal.min_le (λ i:ι α, ⟦⟨α, i.1, i.2⟩⟧) ⟨_, _⟩
-end
-
-lemma ord_eq_min (α : Type u) : ord (#α) =
-  @ordinal.min {r // is_well_order α r} ⟨⟨well_ordering_rel, by apply_instance⟩⟩
-    (λ i, ⟦⟨α, i.1, i.2⟩⟧) := rfl
-
-theorem ord_eq (α) : ∃ (r : α → α → Prop) [wo : is_well_order α r],
-  ord (#α) = @type α r wo :=
-let ⟨⟨r, wo⟩, h⟩ := @ordinal.min_eq {r // is_well_order α r}
-  ⟨⟨well_ordering_rel, by apply_instance⟩⟩
-  (λ i:{r // is_well_order α r}, ⟦⟨α, i.1, i.2⟩⟧) in
-⟨r, wo, h⟩
+quotient.lift_on c (λ α, type (@well_ordering_rel α)) $
+  begin
+    apply eqv_imp_eq_of_eqv_imp_le,
+    rintro α β ⟨e⟩,
+    haveI : is_well_order α (e ⁻¹'o well_ordering_rel),
+      from rel_embedding.is_well_order (rel_iso.preimage e _).to_rel_embedding,
+    calc type (@well_ordering_rel α) ≤ type (e ⁻¹'o well_ordering_rel) :
+      type_well_ordering_rel_le _
+    ... = type well_ordering_rel : (rel_iso.preimage e _).ordinal_type_eq
+  end
+
+@[simp] lemma _root_.ordinal.type_well_ordering_rel (α : Type u) :
```

Comment:
This shouldn't be `simp` for the same reason it wasn't made `simp` in my original PR. Namely:
1. You don't always care about the exact type of this relation, so it's just noise in a lot of circumstances.
2. It breaks a lot of the `bsup` API (though I'm planning on phasing that out for `supr` in the future).